### PR TITLE
[Email] Show and open existing draft when clicking on reply/forward of message

### PR DESCRIPTION
**Pull request by yifanplanet**
# Code changes

- [x] Checks existing draft in thread when clicking on reply/forward, and opens composer with draft if exists.
- [x] Highlights draft in thread when its opened with Composer
- [x]  Removed obsolete property `messageLoadStatus` to simplify `fetchIndividualMessage` function when loading messages and drafts

# Readiness checklist

- [x] Added changes to component `CHANGELOG.md`
- [x] Cypress tests passing?
- [x] New cypress tests added?

# License

I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: components/email/src/Email.svelte [hunk 1]
```diff
@@ -226,7 +226,6 @@
 
   let main: Element;
   let messageRefs: Element[] = [];
-  let messageLoadStatus: string[] = []; // "loading" | "loaded"
```
Comment (yifanplanet) [inline components/email/src/Email.svelte:229]: Remove obsolete property

**Review by philrenaud — APPROVED** [review]
Looks good!

One small UX improvement that could be made here: could we `.focus()` on the composer body when it's opened from fwd/reply buttons? It would save the user a click, or a handful of tabs.

File: components/email/src/Email.svelte [hunk 2]
```diff
@@ -665,40 +675,40 @@
...
       // Don't fetch message when thread is being passed manually
-      if (!_this.thread) {
-        fetchIndividualMessage(
-          draftIndex,
-          activeThread.drafts[draftIndex].id,
-        ).then((res) => {
-          activeThread.drafts[draftIndex].body = res;
+      if (!_this.thread && draft.id) {
+        fetchIndividualMessage(draft.id).then((res) => {
+          draft.body = res;
```
Comment (pooja169usp) [inline components/email/src/Email.svelte:711]: Don't we need to update `activeThread.drafts` here?

**Review by pooja169usp — APPROVED** [review]
Lgtm

File: components/email/src/Email.svelte (hunk 2 above)
Comment (yifanplanet) [reply components/email/src/Email.svelte:711]: Currently all the usages of `dispatchDraftEvent` are having the `draft` parameter as reference.

**Pull request merged**